### PR TITLE
Add tests for offers

### PR DESCRIPTION
**Pull request by arturdm**

### CODE REVIEW

File: sdk/src/integrationTest/resources/raw/offers_get_redemptions.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "data": []
```
Comment (arturdm) [inline sdk/src/integrationTest/resources/raw/offers_get_redemptions.json]: Need real redemptions response here.

File: sdk/src/integrationTest/resources/raw/offers_get_similar_brand_spending.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "data": [],
```
Comment (arturdm) [inline sdk/src/integrationTest/resources/raw/offers_get_similar_brand_spending.json]: Need real response here.

File: sdk/src/integrationTest/resources/raw/offers_get_redemptions.json (hunk 1 above)
Comment (agustk) [reply sdk/src/integrationTest/resources/raw/offers_get_redemptions.json]: I'll send you a redemption payload

File: sdk/src/integrationTest/resources/raw/offers_get_similar_brand_spending.json (hunk 2 above)
Comment (agustk) [reply sdk/src/integrationTest/resources/raw/offers_get_similar_brand_spending.json]: I'll ask for some json from banana stand

File: sdk/src/main/java/com/meniga/sdk/converters/MenigaRedemptionConverter.kt [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import com.meniga.sdk.models.offers.redemptions.MenigaScheduledReimbursement
+import okhttp3.ResponseBody
+import retrofit2.Converter
+import retrofit2.Retrofit
+import java.lang.reflect.Type
+
+/**
+ * Copyright 2017 Meniga Iceland Inc.
+ */
+class MenigaRedemptionConverter(private val gson: Gson) : MenigaConverter() {
```
Comment (agustk) [inline sdk/src/main/java/com/meniga/sdk/converters/MenigaRedemptionConverter.kt]: Is there a reason the converter is called MenigaRedemptionConverter but the only type it handles is MenigaSimilarBrandSpendingDetails?

Comment (arturdm) [reply sdk/src/main/java/com/meniga/sdk/converters/MenigaRedemptionConverter.kt]: Good catch

**Pull request merged**